### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by rainbowflesh** [YaSuenag/SimpleCom#19]
Thx for your great software.

**Comment by YaSuenag** [comment]
Yeah, I'm working for new release (v1.0.1) now, but I cannot commit release schedule because this open source project is my hobby 😎

**Comment by rainbowflesh** [comment]
> Yeah, I'm working for new release (v1.0.1) now, but I cannot commit release schedule because this open source project is my hobby 😎

Excellent! And another thing, I need your help, Im not cpp programmer, and I try build by myself with vs 2022, but its failure, here is error:
```text
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2039	'contains': is not a member of 'std::map<TString,TString,std::less<TString>,std::allocator<std::pair<const TString,TString>>>'	SimpleCom	~\SimpleCom\SimpleCom\SerialDeviceScanner.cpp	105	

```
How can I fix it?

**Comment by YaSuenag** [comment]
Thanks for your report! It seems to be another issue, so I opened #20, and I fixed it! Also I left the cause of error which you saw to its comment.

I keep to open this issue until I ship new release!

**Comment by YaSuenag** [comment]
I released v1.1.0 : https://github.com/YaSuenag/SimpleCom/releases/tag/v1.1.0

Now I've sent [PR to winget-pkgs](https://github.com/microsoft/winget-pkgs/pull/99865) to register this. I will close this issue when I add winget instruction to release page.

**Comment by YaSuenag** [comment]
SimpleCom v1.1.0 has been registered to winget-pkgs, so all done to release process 🎉 - I close this issue.
